src/RedLock.php

```diff
@@ -128,7 +135,7 @@ public function runLocked($resource, $ttl, $closure)
         } catch (Exceptions\ClosureRefreshException $e) {
             return false;
         } finally {
-            if (is_array($lock)) {
+            if (is_object($lock)) {
```

Comment:
In the readme we suggest the caller just tests that this is truthy, should we do the same internally?

Reply:
I think internally it's the same anyway so I can drop the`is_object` check.